spec/features/scheduling/scheduled_publishing_spec.rb

```diff
@@ -64,6 +64,7 @@ def and_a_publish_intent_is_created
 
   def when_the_publishing_job_runs
     travel_to(Time.zone.parse("2019-6-22 9:00"))
+    populate_default_government_bulk_data
```

Comment:
I think ideally we'd just have less feature tests for this and more of the other conditions handled by requests specs, where this would not be a concern. It doesn't seem out of place to do this sort of thing a few times in our feature specs.

I was rather sad when I realised the effect of the travel_to on clearing the cache and that I had to do this.

Reply:
What was the effect of `travel_to` on the cache?

Reply:
What was the effect of `travel_to` on the cache?

Reply:
We cache bulk data for 24 hours. If we travel more than 24 hours after that time in Ruby code the cache will be invalidated. Here we jump from June 21st 0:00am to June 22nd 9:00am